### PR TITLE
0082 timeline

### PR DESCRIPTION
**Pull request by csteamengine**
### Problem / Feature
Timeline looked funky on mobile, squeezed the text into a thin column
### Solution
I switched the Nicholas Cage pic with the HackISU pick, it looked a little more professional than the SuperCage. We can find somewhere else on the site to place him.
### What To Test
Make sure it looks good on different browsers both mobile and desktop
### Reviewers

@REVIEWER

FYI: @KrashLeviathan @csteamengine @regalcat

### CODE REVIEW

File: about/about.html [hunk 1]
```diff
@@ -112,20 +112,16 @@ <h3 class="media-heading">Going public</h3>
                             have our hands full!
                         </div>
                     </li>
-                    <li class="media">
-                        <div class="media-left media-middle">
-                            <img class="media-object" src="../_images/hackisu-logo.png" alt="HackISU Logo" style="height:75px">
-                        </div>
-                        <div class="media-body media-middle text-right">
+                    <li class="media HackISU">
```
Comment (KrashLeviathan) [inline about/about.html]: I'm curious why you added the `HackISU` class name? We don't have a css attribute for that do we?

**Comment by KrashLeviathan** [comment]
I was looking more into the whole layout of the About page, and we might want to switch to a grid-style layout so that on smaller screens the text can jump down below the images. So we don't get this:

aboutpage

File: about/about.html (hunk 1 above)
Comment (csteamengine) [inline about/about.html]: Yeah that was an accident. I originally tried to get it to work with both pictures, which would need its own css. However, I decided to just delete the picture of Nicholas Cage superman. Speaking of Nicholas Cage, Knowing was on TV last night and I like that movie.

**Comment by KrashLeviathan** [comment]
+1 once branch is updated.

**Comment by KrashLeviathan** [comment]
@csteamengine you should switch this page to a grid-style layout (see comment above) and finish this PR before break. Levi is going to update the live website after finals, so we need to try to get these things fixed before then. PS - there's a life-sized bb8 model on display in the toy section of Target.... for inspiration :stuck_out_tongue_winking_eye:

**Comment by csteamengine** [comment]
Haha I know I was looking at it today for a reference

**Comment by frostbyte100** [comment]
Should this be closed now?

**Comment by KrashLeviathan** [comment]
@csteamengine Create a branch on the new **[wdc-website](http://webdev.stuorg.iastate.edu)** repo and create a new merge request. We were not able to remove the `travis-ci` that was holding up further progress on this repo. Sorry for the inconvenience!

**Pull request closed without merging**